src/ResourceManager/EventHub/Commands.EventHub/help/New-AzureRmEventHub.md

```diff
@@ -12,6 +12,12 @@ Creates a new Event Hub.
 
 ## SYNTAX
 
+### EventhubDefaultSet (Default)
+```
+New-AzureRmEventHub [-ResourceGroupName] <String> [-Namespace] <String> [-Name] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
 ### EventhubInputObjectSet
 ```
 New-AzureRmEventHub [-ResourceGroupName] <String> [-Namespace] <String> [-Name] <String>
```

Comment:
ResourceGroupName, Namespace, and Name should be gotten from the InputObject and so shouldn't be in this parameter set.

Reply:
updated the parameter set.